core/vm/logger.go

```diff
@@ -76,6 +77,7 @@ type StructLogger struct {
 	cfg LogConfig
 
 	logs          []StructLog
+	sizeLogs      int
```

Comment:
Is this really necessary? Can't you just use `len(logs)` instead of keeping an additional counter?